tests/test_utils/perf/perf_kit/repeat_and_time.py

```diff
@@ -120,10 +119,10 @@ def monte_carlo(total=10000):
         for _ in range(0, total):
             x_val = random.random() ** 2
             y_val = random.random() ** 2
-            if math.sqrt(x_val + y_val) < 1.0:
+            if x_val + y_val < 1:
```

Comment:
This doesn’t look right?

Reply:
Mathematically if `math.sqrt(x) < 1` then `x < 1` as well, but this is probably useless optimization that hides the original idea. So I have reverted this change (and only replaced `1.0` with `1`).